LiskHQ/lisk-hub
# Fetch all data for Votes tab - Closes #1820

### What issue have I solved?

#1820

### How have I implemented/fixed it?

Created action that receives the votes, how many are show, and filter that fetches delegates data as needed.

### How has this been tested?

Go to `/explorer/accountsV2/2329183154524426169L` since this account has votes for some low ranked delegates.
- Go to votes tab;
- Scroll down and click on show more;
  - A loading state should show up while fetching the new data.
- Reload page and check that filtering by username also triggers the additional fetch.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
